frontend/src/clients/internalApiClient.ts

```diff
@@ -35,3 +35,24 @@ export async function fetchSightings(): Promise<SightingListResponse> {
   const response = await fetch(`https://localhost:5001/`);
   return await response.json();
 }
+
+export async function authenticateLogin(
+  username: string,
+  password: string
+): Promise<boolean> {
+  const details = `${username}:${password}`;
+  const encodedDetails = btoa(details);
+  const authHeader = `Basic ${encodedDetails}`;
+  const response = await fetch(`https://localhost:5001/login`, {
+    method: "GET",
+    headers: {
+      Authorization: authHeader,
```

Comment:
This is very nitpicky, but could we wrap this in speech marks?
```suggestion
      "Authorization": authHeader,
```
The reasoning I can't fully rationalise, but the field name in this case corresponds to a specifically formatted, "human"-readable string in the request (plus I've seen it done that way before).

Basically, the key name here isn't "just code", it's relevant in another context, so the specific formatting and spelling matters ergo string

Reply:
Wrapped Authorization in quotation marks as requested